config/initializers/inflections.rb

```diff
@@ -17,4 +17,5 @@
 
 ActiveSupport::Inflector.inflections do |inflect|
   inflect.acronym 'API'
+  inflect.acronym 'PGDE'
```

Comment:
Maybe use tablename instead so it close to the model

Reply:
Not sure what you mean? This is here so that autoloading tries to load `PGDECourse` instead of `PgdeCourse`.

Reply:
I was referring to 


```
class Mouse < ActiveRecord::Base
  self.table_name = "mice"
end
```

I guess this is something else

Reply:
Yeah, this is something else.